### DISCUSSION TITLE
Stylus

### DISCUSSION THREAD

**Discussion by mustafa-archived** [catppuccin/catppuccin#2041] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Stylus

### Any additional comments?

[Stylus](https://github.com/mustafakhalaf-git/stylus)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
this will not work
in firefox, extension id's are based on the profile (which is different from everyone, mine is 1a64324b-e867-4d57-bd0b-4d933ea793a1), and probably the same with chromium
the user would probably have to provide their own id

**Reply by mustafa-archived** [reply]
We'll get it work eventually (I think a regex that reads the first part and last part of text which are always the same would work)

**Reply by isabelroses** [reply]
> probably the same with chromium the user would probably have to provide their own id

Chromium based browsers, the extension id is set by the webstore
